### DISCUSSION TITLE
Disable supermaven.nvim for large files

### DISCUSSION THREAD

**Discussion by S1M0N38** [LazyVim/LazyVim#4883] [Extra Requests]
I'm using SuperMaven Extra for code completion. It works great, but when I enter a large file, it tries to send the code to the server. The SuperMaven plugin realizes that the file is way too big to be used for code completion, so it avoids generating completion warnings to the user with a message.

![Image](https://github.com/user-attachments/assets/16cf43ab-e704-4d4d-a8ae-ce6b69beca83)

Moreover, a new message “SuperMaven-nvim: File is too large to send to server. Skipping..." appears every time the SuperMaven plugin is triggered (e.g. moving in the file with jk). 

I've noticed that with the introduction of snacks.nvim, LazyVim disables some specific features for large files. I wonder if AI completion plugins such as SuperMaven can be automatically disabled for large files.

**Comment by iamxiaojianzheng** [comment]
Have you solved it ?

**Reply by S1M0N38** [reply]
I think https://github.com/LazyVim/LazyVim/issues/5139 solves it

**Reply by iamxiaojianzheng** [reply]
However, when I updated to the latest lazyvim, this problem still exists.
